docs/arf.md

```diff
@@ -1679,21 +1679,12 @@ Wallet Provider.
 
 A User installs a Wallet Instance because they want to obtain and use
 one or more PIDs and/or attestations. However, there is no requirement
-for a PID Provider or an Attestation Provider to support all certified
+for a PID Provider to support all certified
 Wallet Solutions in the EUDI Wallet ecosystem. This means that the PID
-Provider or an Attestation Provider are not obliged to issue a PID or an
-attestation respectively to any certified Wallet Instance upon the
+Providers are not obliged to issue a PID to any certified Wallet Instance upon the
 request of the User. Instead, they may decide to support only a single
 Wallet Solution, or a limited number of Wallet Solutions.
 
```

Comment:
these sentences below is still useful, no? This is a sentence that is a problem that needs to be removed, not? `Therefore, each PID Provider and Attestation Provider will publish a list of Wallet Solutions that they support.`

```suggestion

Wallet Provider can also publish a list of PID Providers and Attestation Providers
supporting their Wallet Solution. This allows Users to know whether the
Wallet Instance they are about to install will support the PID and the
attestation(s) they wish to use.
```

Reply:
my point is that Attestation Providers should accept all legal eIDAS Wallets

Reply:
I agree.

Reply:
do you expect all wallets to support all attestation providers? I don't and I thought that was the point of this paragraph.

Reply:
ok, talked with Paul and agreed that the assumption here is that if the wallet and the attestation providers support the same technical protocol, they will be able to interoperate. so I am ok removing.